src/helpers/sendFirebaseNotification.ts

```diff
@@ -12,7 +12,7 @@ interface FirebaseNotificationProps {
   body?: string
   postId?: number
   feedId?: number
-  type?: 'entanglement' | 'newpost'
+  type?: 'entanglement' | 'newpost' | 'waitlist'
```

Comment:
I assume it's better to create enum for that, imo.

Reply:
enum is pretty the same in case we use those, no need for it